src/persistence/index.js

```diff
@@ -76,6 +76,105 @@ export class PersistDB {
         return PersistDB.open().then(db => db.openDocument(id))
     }
 
+    /**
+     * Return JS Object of IndexedDB with DB_NAME.
+     */
+    static toJS() {
+        return PersistDB.open().then(async db => {
+            const tx = db.database.transaction(
+                ['states', 'changes', 'contents'], 'readonly')
```

Comment:
I'd prefer this code to be independent from current implementation, for example using database properties such as `IDBDatabase.objectStoreNames`, `IDBDatabase.name`, or `IDBDatabase.version`.

Reply:
I'm not sure about this. It's easier to manage when we do exactly what we want and when we will update database we just have to remember to update also those changes, but I'm open to suggestions ho to make this universal.